docs/api-reference/data-fetching/getStaticProps.md

```diff
@@ -134,47 +137,7 @@ export async function getStaticProps(context) {
 }
 ```
 
-Redirecting at build-time is currently not allowed and if the redirects are known at build-time they should be added in [`next.config.js`](/docs/api-reference/next.config.js/redirects.md).
```

Comment:
Just to confirm, removing this was intentional?

Reply:
It was duplicated, so i removed it here 😊